woocommerce/woocommerce
# Fixed user cap required to view the order count indicator

### All Submissions:






### Changes proposed in this Pull Request:



Checking for `manage_woocommerce` is not correct, since Orders post type uses [`shop_order` capability type](https://github.com/woocommerce/woocommerce/blob/dfcc5e16e517a693cc7bcdb256e744c5a9bf544d/includes/class-wc-post-types.php#L399).

Closes #25561.

### How to test the changes in this Pull Request:

It's necessary to also apply the changes from #26877 in order to see the icon.

1. Remove the `manage_woocommerce` capability from shop manager role or create a new role with [`shop_order` capability type](https://github.com/woocommerce/woocommerce/blob/ee119e0a7ed24cf8c83983ff013bd188e3081d19/includes/class-wc-install.php#L1122-L1148).
2. View the WordPress dashboard
3. Orders menu item has no indicator

### Other information:




### Changelog entry

> Fix - Fixed user capability required to view the order count indicator.
